ui/src/index.css

```diff
@@ -2,6 +2,92 @@
 @tailwind components;
 @tailwind utilities;
 
+h1 {
+  font-size: 1.5rem;
+  line-height: 3rem;
+  font-weight: 700;
+  margin-top: 0;
+  margin-bottom: 1rem;
+}
+
+/* Basic link style */
+a {
+  color: #0645ad; /* A shade of blue for the link */
+  text-decoration: none; /* Removes underline from links */
+  transition: color 0.3s ease-in-out; /* Smooth color transition */
+  text-decoration: underline;
+}
+
+/* Hover state */
+a:hover {
+  color: #0a58ca; /* A darker shade of blue for hover state */
+}
+
+/* Active state */
+a:active {
+  color: #03396c; /* An even darker shade for active/clicked state */
+}
+
+/* Visited state */
+a:visited {
+  color: #4b2e83; /* A purple color for visited links */
+}
```

Comment:
This stying is going to affect the entire app, unless these changes should be across the board, you need to scope these or rename